docs/userguide/execution.rst

```diff
@@ -39,7 +39,7 @@ parameters include access keys, instance type, and spot bid price
 Parsl currently supports the following providers:
 
 1. `parsl.providers.LocalProvider`: The provider allows you to run locally on your laptop or workstation.
-2. `parsl.providers.CobaltProvider`: This provider allows you to schedule resources via the Cobalt scheduler. **This provider is deprecated and will be removed by 2024.04**.
+2. `parsl.providers.CobaltProvider`: This provider allows you to schedule resources via the Cobalt scheduler. This provider is DEPRECATED and will be removed by 2024.04
```

Comment:
Capitalizing deprecated looks reasonable. I prefer the **bold** line because it easily grabs attention, so i recommend adding the `**`s back.